node-manager/src/nodemanager.rs

```diff
@@ -73,7 +73,7 @@ impl NodeIdentity {
 #[derive(Serialize, Deserialize, Clone, Eq, PartialEq)]
 #[wasm_bindgen]
 pub struct MutinyInvoice {
-    bolt11: String,
+    bolt11: Option<String>,
```

Comment:
Just realized I didn't have to put `Option` here and could have put `""`. 

@benthecarman  @futurepaul  thoughts on returning `Option<String>` vs empty string?

Reply:
None is more explicit than an empty string so we should use that